rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -384,23 +384,6 @@ error_code sceNpTrophyCreateContext(vm::ptr<u32> context, vm::cptr<SceNpCommunic
 	// append the commId number as "_xx"
 	std::string name = fmt::format("%s_%02d", name_sv, commId->num);
 
-	// open trophy pack file
-	std::string trophy_path = vfs::get(Emu.GetDir() + "TROPDIR/" + name + "/TROPHY.TRP");
-	fs::file stream(trophy_path);
-
-	if (!stream && Emu.GetCat() == "GD")
-	{
-		sceNpTrophy.warning("sceNpTrophyCreateContext failed to open trophy file from boot path: '%s'", trophy_path);
-		trophy_path = vfs::get("/dev_bdvd/PS3_GAME/TROPDIR/" + name + "/TROPHY.TRP");
-		stream.open(trophy_path);
-	}
-
-	// check if exists and opened
-	if (!stream)
-	{
-		return {SCE_NP_TROPHY_ERROR_CONF_DOES_NOT_EXIST, trophy_path};
```

Comment:
It might be read-only error checking skip, you simply need to change the path to one that doesn't exist to test.

Reply:
We used your trophy test for this.
In the last run i asked neko to remove the directory and the test still ran fine. It should have ended due to ENSURE_OK i guess.
You can run the test again to confirm if you want.

Anyway, Lego Harry Potter doesn't use the read-only flag but still fails on the same check.

Reply:
Idk if he tested both options ☺️

Reply:
that's what I'm asking..

Reply:
(omitted some log spam)
sceNpTrophyGetTrophyUnlockState(error 0x80022904, line=118)
sceNpTrophyGetRequiredDiskSpace(error 0x8002291b, line=121)
sceNpTrophyGetTrophyInfo(error 0x80022904, line=127)
"sceNpTrophyRegisterContext(context, handle, complexCallback, &var, 0)" Failed at line 148! (error=0x8002291b)

Reply:
so it seems the CreateContext doesn't care about the trophy file

Reply:
okay

Reply:
The original code is 8 years old. Who knows what hacks had been in place to get some games going xD